test/checks/aria/required-children.js

```diff
@@ -273,6 +315,24 @@ describe('aria-required-children', function() {
 		);
 	});
 
+	it('should pass one existing aria-owned child when one required', function() {
+		var params = checkSetup(
+			'<ul id="target"  role="tablist"><li role="tab">Tab 1</li><li role="tab">Tab 2</li></ul>'
+		);
+		assert.isTrue(
+			checks['aria-required-children'].evaluate.apply(checkContext, params)
+		);
+	});
+
+	it('should pass one existing aria-owned child when one required', function() {
```

Comment:
Probably should name these test cases so they are different. I had to look at the code to see what exactly made them different.